microprofile-jwt/src/test/java/org/jboss/eap/qe/microprofile/jwt/activation/ActivationPropertiesUnsetTest.java

```diff
@@ -0,0 +1,75 @@
+package org.jboss.eap.qe.microprofile.jwt.activation;
+
+import org.jboss.arquillian.container.test.api.Deployer;
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.arquillian.test.api.ArquillianResource;
+import org.jboss.eap.qe.microprofile.jwt.testapp.jaxrs.NonJwtJaxRsTestApplication;
+import org.jboss.eap.qe.microprofile.jwt.testapp.jaxrs.SecuredJaxRsEndpoint;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+@RunWith(Arquillian.class)
+public class ActivationPropertiesUnsetTest {
+
+    private static final String NO_VALUE_DEPLOYMENT_NAME = "PROPERTIES_UNSET_DEPLOYMENT";
+    private static final String OTHER_VALUE_DEPLOYMENT_NAME = "OTHER_VALUE_DEPLOYMENT";
+
+    @Deployment(name = NO_VALUE_DEPLOYMENT_NAME, managed = false)
+    public static Archive<?> createDeploymentNoValue() {
+        return ShrinkWrap
+                .create(WebArchive.class,
+                        ActivationPropertiesUnsetTest.class.getSimpleName() + NO_VALUE_DEPLOYMENT_NAME + ".war")
+                .addClass(SecuredJaxRsEndpoint.class)
+                .addClass(NonJwtJaxRsTestApplication.class)
```

Comment:
Would you evaluate whether to join the two `addClass()` statements, here and for all following `ShrinkWrap` archives generation? We've been following this practice in recent PRs.

Reply:
I am not sure I follow. Can you please provide me with an example?

Reply:
Sure:
```
.addClass(SecuredJaxRsEndpoint.class, NonJwtJaxRsTestApplication.class)
```

Reply:
To be honest, I like it better the way I did it first. Though if you want me to change it I have no objections...

Reply:
I definitely wouldn't force you to do so :)
I was just trying and letting you know that, in a recent review, I was asked to do so and thus it seemed reasonable to me to keep the same code approach. 
Anyway, if you have some rationale for it (e.g. to emphasize the difference in using a non-JWT annotated class for JAX-RS application, I am fine with that. I'll leave this decision to you.

Reply:
Thank you, I understand why are pointing this out. To be honest, I have no objective arguments supporting my decision or arguments against my decision regarding using multiple method calls or one method call with multiple arguments. Maybe one (and this is just my POV) is that calling multiple methods makes it look more tidy and clear. Though there are probably other arguments which I'm failing to see.

If you are not against it, I would leave this as-is...

Reply:
Thanks. As I stated earlier, the reasoning behind my comment was to have the same common approach to coding style for methods building a ShrinkWrap archive. 
That said, your reasoning about code being more tidy and clean makes sense to me too in this very case.
So let's agree on leaving this as-is.